test/integration/test_keyprotect.py

```diff
@@ -257,6 +257,54 @@ def test_instance_policies(self):
                 )
                 self.assertFalse(resource["policy_data"]["attributes"]["enforce_token"])
 
+    def test_set_keyring(self):
+        # create a key to be used for test, the key will by default be created in the 'default' key ring
+        self.key = self.kp.create(name="test-key", root=True)
```

Comment:
Why are you setting `self.key` rather than just using the key object directly? I don't see where you would be using it outside this function...

Reply:
Fixed, thanks!